### PR TITLE
more detailed example in the README

### PR DESCRIPTION
**Pull request by archiloque**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,9 +128,28 @@ require 'queue_classic'
...
+
+  # A job contain these attributes:
+  # :id Integer, the job id
+  # :method String, containing the object and method
+  # :args String, the arguments
+  # :q_name String, the queue name
+  # :scheduled_at Time, the scheduled time if the job was scheduled
+
+  # Execute the job using the methods and arguments
+  def call(job)
```
Comment (jipiboily) [inline README.md]: That's a relic from the previous PR, right?

Comment (archiloque) [inline README.md]: It's a part of the previous PR but translated for the job-is-a-hash, I just added a comment to tell explicitely that job is a Hash

**Comment by jipiboily** [comment]
Just looked quickly and it looks good...

**Comment by jipiboily** [comment]
Thanks!

**Pull request merged**